src/util/FormController.js

```diff
@@ -47,7 +47,7 @@ function (Okta, FormType, Toolbar, FormUtil, BaseLoginController, BaseLoginModel
           FormUtil.applyShowWhen(this.last(), child.viewOptions && child.viewOptions.showWhen);
           break;
         case FormType.VIEW:
-          this.add(child.viewOptions.View);
+          this.add(child.viewOptions.View, child.addOptions);
```

Comment:
```
if (!webauthn.isAvailable()) {
  result.push(
    FormType.View(
      { View: WindowsHelloErrorMessageView },
      { selector: '.o-form-error-container' }
    )
  );
}
```

Otherwise `WindowsHelloErrorMessageView` will be added at the end of the form, not inside the error-container.